### PR TITLE
Upgrade to Sf3.4

### PR DESCRIPTION
**Pull request by pdias**
Make services public by default

### CODE REVIEW

File: Resources/config/assetic.xml [hunk 1]
```diff
@@ -25,6 +25,9 @@
     </parameters>
 
     <services>
+        
+        <defaults public="true" />
```
Comment (Jean-Beru) [inline Resources/config/assetic.xml]: `<defaults public="true" />` is not compatible with SF2.3. But set each service can be set as public.

**Comment by hackzilla** [comment]
My gut says it's because PHP Unit is a newer version.

```
Fatal error: Class 'PHPUnit_Framework_TestCase'
```

I believe it changed to \PHPUnit\Framework\TestCase

**Comment by Nek-** [comment]
This doesn't fix everything for 3.4. Also, it needs to be squash.

**Comment by ajtis** [comment]
+1

**Pull request closed without merging**